.github/workflows/ci.yml

```diff
@@ -28,3 +28,30 @@ jobs:
 
       - name: Overall test
         run: cargo test
+      
+  dvm-rpc-test:
+    if: "contains(toJSON(github.event.head_commit.message), '[skip dvm]')"
```

Comment:
Should this be something like `!contains(..., '[skip dvm]')`?

Reply:
A bit weird here. The second action workflow will be triggered when `if: "!contains(toJSON(github.event.head_commit.message), '[skip dvm]')"`. But, without `!`, it works well.

Reply:
I have tried hundred times with the head_commit flag to skip the DVM test for specific situations. Just ignore this feature at present.